yellowbrick/features/pcoords.py

```diff
@@ -174,15 +177,14 @@ class ParallelCoordinates(DataVisualizer):
     shuffle : boolean, default: True
         specifies whether sample is drawn randomly
 
-    color : list or tuple, default: None
-        optional list or tuple of colors to colorize lines
-        Use either color to colorize the lines on a per class basis or
-        colormap to color them on a continuous scale.
+    colors : list or tuple, default: None
```

Comment:
Noticing a few places here where we seem to be deprecating the old `color` param for `colors` &mdash; which I'm at peace with (possibly no one but me and my Visual Analytics students ever used these 🤣) but maybe just something we should note as backwards incompatible in our endless release notes doc...